### PR TITLE
Enable yast2_firstboot in textmode

### PR DESCRIPTION
**Pull request by JRivrain**
In case we don't have an X server available, firstboot uses ncurses, this new test will cover it.
Also deleting a soft-failure for a closed bug bsc#1173298

- Related ticket: https://progress.opensuse.org/issues/73471
- Needles: https://gitlab.suse.de/openqa/os-autoinst-needles-sles/-/merge_requests/1476
- Verification run: http://waaa-amazing.suse.cz/tests/13911#

### CODE REVIEW

File: tests/installation/yast2_firstboot.pm [hunk 1]
```diff
@@ -73,12 +74,6 @@ sub firstboot_root {
 }
 
 sub firstboot_hostname {
-    if (check_screen('bsc1173298', 30)) {
```
Comment (rwx788) [inline tests/installation/yast2_firstboot.pm:76]: Could you please move this change to separate commit, as we remove workaround here.

Comment (JRivrain) [reply tests/installation/yast2_firstboot.pm:76]: done
